src/Symfony/Component/Process/Tests/OutputMemoryLimitProcess.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Tests;
+
+use Symfony\Component\Process\PhpExecutableFinder;
+use Symfony\Component\Process\PhpProcess;
+use Symfony\Component\Process\PhpSubprocess;
+use Symfony\Component\Process\Process;
+
+require \dirname(__DIR__).'/vendor/autoload.php';
```

Comment:
This currently causes an issue because that file does not exist on the mono repository. However, we already do have the same code here: https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Process/Tests/ErrorProcessInitiator.php
That's actually a bug. That test should fail already and get fixed. Not sure how but I've added a an issue here: https://github.com/symfony/symfony/issues/48486
I will adapt the PR to the solution found to that ticket :)

Reply:
please check #51207